src/CoreFoundation/CFString.cs

```diff
@@ -92,13 +92,13 @@ public override string ToString ()
 		}
 	}
 
-#if XAMCORE_4_0
+#if NET
 	// nothing is exposed publicly
-	internal
+	internal static class CFObject {
```

Comment:
Ideally (it would reduce metadata) the p/invoke would be moved (into `NativeObject`) but that requires changes in many places. The good news is that can be done later since it won't be a breaking change.